libjxl/src/main/java/fr/oupson/libjxl/JxlDecoder.kt

```diff
@@ -0,0 +1,13 @@
+package fr.oupson.jxlviewer
```

Comment:
Kotlin is useless in this case and only add dependancies.

Reply:
Converted the code to Java (and also made it a static method instead of requiring a dummy object)